src/check_plugin_vulnerability.py

```diff
@@ -28,12 +28,20 @@
 import requests
 from sys import exit
 
-SCRIPT = (
-    'def warnings_monitor = new jenkins.security.UpdateSiteWarningsMonitor()\n'  # NOQA E501 Don't wrap because of groovy code
-    'def plugin_vulnerabilities = warnings_monitor.getActivePluginWarningsByPlugin()\n'  # NOQA E501 Don't wrap because of groovy code
-    'def critical_plugins = plugin_vulnerabilities.keySet()*.longName\n'
-    'print(new groovy.json.JsonBuilder(critical_plugins))'
-)
+SCRIPT = """
+    import jenkins.security.*
```

Comment:
It's good practice to use explicit non grouped imports:
```suggestion
    import jenkins.security.UpdateSiteWarningsConfiguration
    import jenkins.security.ExtensionList
```